inc/customizer/layouts.php

```diff
@@ -564,6 +566,27 @@ public function get_current_layout( $post = null ) {
 
 	}
 
+	/**
+	 * Magic getter for `$colors` and `$color_schemes` properties.
+	 *
+	 * @since  1.0.0
+	 *
+	 * @param  string $name Color key or color scheme slug name.
+	 *
+	 * @return string Return the specified property within the `Primer_Customizer_Colors` class.
+	 */
+	public function __get( $name ) {
+
+		if ( ! in_array( $name, array( 'layouts', 'default', 'meta_box', 'page_widths' ), true ) ) {
+
+			return false;
+
+		}
+
+		return $this->$name;
```

Comment:
This can be a two-liner:

```php
$properties = array( 'layouts', 'default', 'meta_box', 'page_widths' );

return in_array( $name, $properties, true ) ? $this->$name : false;
```